.github/workflows/bump_version.yml

```
@@ -0,0 +1,63 @@
+name: Bump Version
+
+on:
+  workflow_dispatch:
+    inputs:
+      newVersion:
+        description: "Version name"
+        required: true
+        type: string
+
+jobs:
+  bump_version:
+    name: Bump version
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout the latest code
+        uses: actions/checkout@v2
```

I think we should use the [latest version](https://github.com/actions/checkout) 🤔 
```suggestion
        uses: actions/checkout@v3
```